src/main/java/lotto/domain/Ball.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+public class Ball{
+    private static final String NUMBER_MATCHES = "-?[0-9]+";
+    public static final int MINIMUM_NUMBER = 1;
+    public static final int MAXIMUM_NUMBER = 45;
+    private static final String ERROR_ONLY_NUMBER = "숫자를 입력해주세요!";
+    private static final String ERROR_LOTTO_NUMBER = "로또 숫자가 아닙니다";
+
+    private final int number;
+
+    public Ball(String number) {
+        checkValidValue(number);
+        this.number = Integer.parseInt(number);
+    }
+
+    private void checkValidValue(final String value) {
+        if (isBlank(value) || !isNumber(value)) {
+            throw new IllegalArgumentException(ERROR_ONLY_NUMBER);
+        }
+        int number = Integer.parseInt(value);
+        checkLottoNumber(number);
+    }
+
+    private boolean isBlank(final String number) {
+        return number == null || number.isEmpty();
+    }
+
+    private boolean isNumber(final String value) {
+        return value.matches(NUMBER_MATCHES);
+    }
```

Comment:
요 부분도 작은 팁이고 사람마다 취향이 다를 수 있어요.  

```suggestion
        if (isBlank(value) || isNotNumber(value)) {
            throw new IllegalArgumentException(ERROR_ONLY_NUMBER);
        }
        int number = Integer.parseInt(value);
        checkLottoNumber(number);
    }

    private boolean isBlank(final String number) {
        return number == null || number.isEmpty();
    }

    private boolean isNotNumber(final String value) {
        return !value.matches(NUMBER_MATCHES);
    }
```

위와 같이 메서드를 사용할 때에 !를 붙여서 부정임을 표현하는가 Not을 붙여서 표현하는가에 대한 차이랄까요?!

마루는 어떤쪽이 좋으신가요? 그리고 그 이유는요?

Reply:
저도 보통은 isNotNumber로 쓰는 편인데, 제 페어분께서 만약에 숫자인지를 체크해야한다면 !isNotNumber가 되니까 이중부정이 뭔가 어색할 수 있다고 하시더고요..! 생각해보니 진짜 그게 어색한 느낌이 들어서 저도 동의하고 isNumber로 고쳤었는데, 다시 생각해보니 옆에 있는 isBlank와 비슷한 흐름을 가져가려면 isNotNumber를 써야할것도 같아서ㅋㅋㅋ 어떻게 쓰는게 조금 더 좋은 방법일까요..?!

Reply:
이 질문을 드린 이유는 취향이 갈리기 때문이에요 😇

이처럼 어느 한 쪽을 선택해야 하는 확실한 근거가 없는 문제에 대해서는 같이 협업하는 사람들과 충분한 시간을 가지고 토의한 후에 결정하는 편이 좋답니다! 마루와 페어는 그렇게 고민하고 결정해주신거 같아서 좋아요!

마루가 질문 주신 어떤 게 더 좋은 방법인지에 대해서는 저도 잘 모르겠다는 답변을 드릴게요 😉

(제 취향은 `isNotNumber`를 만들고 만일 `isNumber`가 필요한 때가 오면 따로 `isNumber` 메서드도 만들어줄 거 같아요)